attributes/default.rb

```diff
@@ -43,7 +43,7 @@
 default['ndb']['MaxDiskWriteSpeedOtherNodeRestart']   = "50M"
 default['ndb']['MaxDiskWriteSpeedOwnRestart']         = "200M"
 default['ndb']['MinDiskWriteSpeed']                   = "10M"
-default['ndb']['DiskSyncSize']                        = "4M"
+default['ndb']['DiskSncSize']                        = "4M"
```

Comment:
Accidentally changed this attribute? We don't define or use anywhere `DiskSncSize`.
We do use `DiskSyncSize` which you moved below.

Reply:
Thanks for spotting this.